pennylane/operation.py

```diff
@@ -3012,6 +3013,58 @@ def convert_to_opmath(op):
     return op
 
 
+def convert_to_legacy_H(op):
```

Comment:
I'm not sure we want this to be a user facing function that is [visible in the docs](https://xanaduai-pennylane--5309.com.readthedocs.build/en/5309/code/api/pennylane.operation.convert_to_legacy_H.html?highlight=convert_to_legacy_h#pennylane.operation.convert_to_legacy_H). Is there a usecase for this? My expectation is that users just enable or disable opmath.

Reply:
I agree. This function is meant for internal use only. I'll make it invisible in the docs.